tamr_client/dataset/__init__.py

```diff
@@ -0,0 +1,11 @@
+# See https://github.com/python/mypy/issues/5759
```

Comment:
You'll want to start this file with:

```python
# flake8: noqa
```

to turn off linting for this file (since it will complain about importing a bunch of things that aren't used within this file)

See https://github.com/Datatamer/tamr-client/blob/master/tamr_client/__init__.py#L1